src/app/child-dev-project/children/model/child.ts

```diff
@@ -57,12 +57,12 @@ export class Child extends Entity {
   })
   projectNumber: string;
   @DatabaseField({
-    dataType: "date-only",
+    dataType: "date-of-birth",
```

Comment:
is this even necessary if the TypeScript type is `DateOfBirth` --> could infer the schema dataType?

Reply:
Generally, nice idea. But it has one issue: Classes are renamed in the production build (we had the same issue with the `@DynamicComponent` annotation). Therefore, a static member like the `ENTITY_TYPE` would need to be set on the class to have a fixed name. But this can of course be a option to make this cleaner.

Reply:
oh, right! not sure it's worth setting an ENTITY_TYPE - seems a bit counter-intuitive, so we could also stick with the current way?

Reply:
I saw you had already solved it. Actually nice! :100: